contracts/governance/Governor.sol

```diff
@@ -578,15 +578,8 @@ abstract contract Governor is Context, ERC165, EIP712, Nonces, IGovernor, IERC72
         address voter,
         bytes memory signature
     ) public virtual returns (uint256) {
-        bool valid = SignatureChecker.isValidSignatureNow(
-            voter,
-            _hashTypedDataV4(keccak256(abi.encode(BALLOT_TYPEHASH, proposalId, support, voter, _useNonce(voter)))),
-            signature
-        );
-
-        if (!valid) {
-            revert GovernorInvalidSignature(voter);
-        }
+        bytes memory rawSignatureDigestData = abi.encode(BALLOT_TYPEHASH, proposalId, support, voter, nonces(voter));
+        _validateVoteSignature(voter, proposalId, signature, rawSignatureDigestData);
```

Comment:
We could do something like

```suggestion
        _validateVoteSignature(
            voter, 
            proposalId, 
            signature, 
            abi.encode(BALLOT_TYPEHASH, proposalId, support, voter), // encodedBeforeNonce
            "", // encodedAfterNonce
        );
```

And let `_validateVoteSignature` fill the nonce using `abi.encodePacked(before, nonce(...), after)`

Reply:
From our discussion: 
- use an offset value to replace the hardcoded 0xA0